spec/javascripts/NewService/NewServiceForm.spec.jsx

```diff
@@ -12,49 +18,23 @@ const props = {
   providerAdminServiceDiscoveryServicesPath: 'my-path',
   adminServicesPath: 'my-other-path'
 }
-const discoveryNotUsableProps = {
-  ...props,
-  isServiceDiscoveryUsable: false
-}
 
 it('should render itself', () => {
   const wrapper = mount(<NewServiceForm {...props}/>)
   expect(wrapper.find('#new_service_source').exists()).toEqual(true)
   expect(wrapper.find(`input[name='source']`).length).toEqual(2)
 })
 
-it('should render new Service Manual form', () => {
+it('should render the correct form depending on which mode is selected', () => {
   const wrapper = mount(<NewServiceForm {...props}/>)
-  expect(wrapper.find('#new_service').exists()).toEqual(true)
-  expect(wrapper.find('#service_discovery').exists()).toEqual(false)
-})
 
-// TODO: remove `skip` when this is merged: https://github.com/airbnb/enzyme/pull/2008
-it.skip('should render new Service Discovery form when click on input', () => {
-  const wrapper = mount(<NewServiceForm {...props}/>)
-  wrapper.find('#source_discover').simulate('click')
+  wrapper.find('#source_discover').simulate('change')
```

Comment:
As `simulate()` is being deprecated you could call `onChange` from props()

Reply:
Yes you're right! 👍